sections/faqs/extend-and-styled-difference.js

```diff
@@ -0,0 +1,25 @@
+import md from 'components/md'
+
+const ExtendAndStyled = () => md`
+  ## What is the difference between \`Comp.extend\` and \`styled(Comp)\`?
+
+  ### Behind the scenes
+  The \`styled()\` factory generates new component styles with a new class. The classnames are
+  then passed to the React component via the \`className\` prop.
+  Calling \`extend\` creates new component styles by extending the old one, and thus doesn't
+  generate two classes for a single component. (\`styled()\` factory does that)
+
+  ### Which one should you use
+  We recommend to use \`.extend\` where possible for internal \`StyledComponents\`,
+  but the \`styled()\` factory for external components and cross-file containers/components
+  where it isn't clear what it's going to be when writing the wrapper.
+
+  > Wrapping a container is not recommended. It should always be attempted to wrap
+  > presentational and structural components only ince cleaner logic can be achieved when
```

Comment:
~and structural~

This is my recommendation (Although I know that it's sometimes unavoidable) since `styled()` in itself creates a presentational wrapper. This wrapper would blur the lines between presentational and structural components.

A `styled()` call on your own structural component would thus only be applied to either the outermost element, any element (especially bad), or use very specific selectors, the latter being what anyone definitely do not want to see in CSS-in-JS.

Applying this is a powerful constraint since it forces you to move presentational logic/variants as low as possible, which makes the resulting structure easier to understand. For instance, an "InputText" component is expected to always behave in a certain way, but let's say in some cases it's expected to look differently. Now, you might be tempted to just wrap it for that case, but isn't it better to introduce a prop for a variating look? That would mean that this one component "stays true" to being a base component, from a styleguide perspective.

It might just be a mixup here, since the next part also refers to containers, not structural components, right?